src/cpp/common/version_cc.cc

```diff
@@ -19,7 +19,7 @@
 #include <string>
 
 #include <grpcpp/grpcpp.h>
-#include <grpcpp/support/config.h>
```

Comment:
This is an autogenerated file, the template would need to be changed instead. But it is unrelated to this PR regardless, so should be rolled back.

Reply:
Restored.